docs/rest-api.md

```diff
@@ -84,12 +89,31 @@ API Version | HTTP Method | Request URL | Request Body
 `1.0-preview` | `POST` | `https://<instance-name>.service.signalr.net/api/v1-preview/hub/<hub-name>/user/<user-id>` | `{ "target":"<method-name>", "arguments":[ ... ] }`
 `1.0` | `POST` | `https://<instance-name>.service.signalr.net/api/v1/hubs/<hub-name>/users/<user-id>` | Same as above
 
+<a name="add-connection-to-group"></a>
+### Add a connection to a group
+
+API Version | HTTP Method | Request URL
+---|---|---
+`1.0` | `PUT` | `https://<instance-name>.service.signalr.net/api/v1/hubs/<hub-name>/groups/<group-name>/connections/<connection-id>`
+`1.0` | `PUT` | `https://<instance-name>.service.signalr.net/api/v1/hubs/<hub-name>/connections/<connection-id>/groups/<group-name>`

```

Comment:
why add one pattern for just switch parameter order, would hubs also be movable? seems not useful information here. Same for the rest.

Reply:
hub isn't movable.
I want to tell readers only the listed URLs are valid